### PR TITLE
Coveo for Sitecore V4 Cloud Road to production

### PR DESCRIPTION
**Pull request by SLangevin**
New blog post

### CODE REVIEW

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+Version 4 was released this spring and allowed Coveo for Sitecore’s users to move their index to the cloud, reducing maintenance effort and opening the way for the advanced cloud features, such as Reveal machine learning and the query pipelines.
+Integrating Coveo for Sitecore to your Sitecore solution is now easier than ever, you can download the package directly on our [site](http://www.coveo.com/en/solutions/coveo-for-sitecore/download) and follow the installation wizard in Sitecore.
+<!-- more -->
+This is great to try the product with a trial organization, but how do I manage a paid license?
+I received several questions in the past few months about environment setup and license management, so I will try to clarify in this post.
+
+## What do I get when I buy?
+
+All Coveo for Sitecore cloud plans offer a Production and Sandbox cloud organization.The owner of the organizations is the recipient of the license.
```
Comment (malaporte) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: Missing space after `.`

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Version 4 was released this spring and allowed Coveo for Sitecore’s users to move their index to the cloud, reducing maintenance effort and opening the way for the advanced cloud features, such as Reveal machine learning and the query pipelines.
+Integrating Coveo for Sitecore to your Sitecore solution is now easier than ever, you can download the package directly on our [site](http://www.coveo.com/en/solutions/coveo-for-sitecore/download) and follow the installation wizard in Sitecore.
+<!-- more -->
+This is great to try the product with a trial organization, but how do I manage a paid license?
+I received several questions in the past few months about environment setup and license management, so I will try to clarify in this post.
+
+## What do I get when I buy?
+
+All Coveo for Sitecore cloud plans offer a Production and Sandbox cloud organization.The owner of the organizations is the recipient of the license. 
+The Coveo license team will rarely accept to change the owner for you due to legal and security concerns. This mean that you, or your client, will need to manage access to the organizations.
```
Comment (malaporte) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: ? that sounds weird. Why exactly? Can't we set an owner in the admin (dunno)? But it'd make sense right?

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  bio:  Coveo for Sitecore Solution Architect
+  image: slangevin.png
+---
+Version 4 was released this spring and allowed Coveo for Sitecore’s users to move their index to the cloud, reducing maintenance effort and opening the way for the advanced cloud features, such as Reveal machine learning and the query pipelines.
+Integrating Coveo for Sitecore to your Sitecore solution is now easier than ever, you can download the package directly on our [site](http://www.coveo.com/en/solutions/coveo-for-sitecore/download) and follow the installation wizard in Sitecore.
+<!-- more -->
+This is great to try the product with a trial organization, but how do I manage a paid license?
+I received several questions in the past few months about environment setup and license management, so I will try to clarify in this post.
+
+## What do I get when I buy?
```
Comment (malaporte) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: This header is pretty generic considering the content that follows. Maybe add some more general info? Or change the header?

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+This page will help you create a group with access to everything except groups and members control. The decision to remove other privileges mostly depend on your security policy. 
+For example, you might want to give read access to the Usage Analytics or Index Content on the sandbox organization, but remove that privilege in production.
+
+## Where do I code? Where do I test?
+
+As a best practice, I would tend to limit my sandbox organization for testing only. For development, you can whether purchase an additional organization, or use trial organizations.
+This is also a good strategy when multiple developers are coding on different copies of your instance. Each of them can have its own organization to code and test against.
+The trial organizations are only valid for one month, but creating a new one is one easy step. Simply go to the Sitecore Control Panel and choose Configuration in the Coveo Search Menu. From there you can use the Coveo Cloud Organization feature to connect to Coveo cloud. Once you are redirected back to Sitecore, you will be given the option to create a new organization. This new org will not be configured, but rebuilding your indexes will do the trick.
```
Comment (malaporte) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: still cost money y'know :)

Any particular reason why the sandbox org can't be shared?

Comment (SLangevin) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: Partner VS clients <= client might not want partner to access their orgs.

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md (hunk 3 above)
Comment (SLangevin) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: Changed

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md (hunk 1 above)
Comment (SLangevin) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: Done

File: _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+title: "Coveo for Sitecore V4 Cloud - The Road to Production"
+tags: [Sitecore, Cloud, Coveo for Sitecore, Environment]
+
+author:
+  name: Simon Langevin
+  bio:  Coveo for Sitecore Solution Architect
+  image: slangevin.png
+---
+Version 4 was released this spring and allowed Coveo for Sitecore’s users to move their index to the cloud, reducing maintenance effort and opening the way for the advanced cloud features, such as Reveal machine learning and the query pipelines.
+Integrating Coveo for Sitecore to your Sitecore solution is now easier than ever, you can download the package directly on our [site](http://www.coveo.com/en/solutions/coveo-for-sitecore/download) and follow the installation wizard in Sitecore.
```
Comment (francoislg) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: IMO, we can't guess the subject of the full article using only the two paragraphs there. The next two lines capture better the "why should I click Read More...". So I would either:

* Put the two next lines here in the excerpt
* Put the current excerpt in the text and add a new intro

Comment (SLangevin) [inline _posts/2016-11-14-coveo-for-sitecore-v4-cloud-road-to-production.md]: I added the lines in the excerpt. If it end up being too long, I will create a new one.

**Pull request merged**